src/fx.js

```diff
@@ -337,6 +337,8 @@ SVG.FX = SVG.invent({
 
     // set/get the speed of the animation
   , speed: function(spd){
+      if (spd === 0) return this.pause()
```

Comment:
forgot to mention this: I would rename `spd` to `_speed` followig the convention of `_target`. But that is up to discussion